custom/icds_reports/migrations/sql_templates/database_views/kpi_api.sql

```diff
@@ -0,0 +1,57 @@
+DROP VIEW IF EXISTS kpi_api_view CASCADE;
+CREATE VIEW kpi_api_view AS
```

Comment:
well. kpi(key performance indicators) is pretty general name. Would be great to make it specific like pmo_api_view. something like that.

Reply:
done